main.py

```diff
@@ -23,18 +33,38 @@ def count_error(data: list[float, ...], *, check_rude_values: bool = False):
     return average, squared_error
 
 
-def delete_rude_values(data: list[float, ...], squared_error: float, average: float):
+def delete_rude_values(data: list[float, ...], squared_error: float, average: float) -> tuple[list[float, ...], bool]:
+    """Delete rude values from data.
+
+    Args:
+        data (list[float, ...]): data to delete rude values from.
+        squared_error (float): squared error.
+        average (float): average error.
+
+    Returns:
+        tuple[list[float, ...], bool]: data without rude values and whether rude values were found.
+    """
     found_rude_value = False
 
     for i in data:
+        # TODO(TheCrabilia): extract 2.29 to constant. What the name should be?
```

Comment:
It is a rude value coefficient, which changes according to the measurement count